clientv3/op.go

```diff
@@ -111,6 +136,24 @@ func OpPut(key, val string, opts ...OpOption) Op {
 	return ret
 }
 
+func opWatch(key string, opts ...OpOption) Op {
+	ret := Op{t: tRange, key: []byte(key)}
+	ret.applyOpts(opts)
+	switch {
+	case ret.end != nil && !reflect.DeepEqual(ret.end, getPrefix(ret.key)):
+		panic("only supports single keys or prefixes")
```

Comment:
@heyitsanthony Could you check this part? `ret.end != nil && !reflect.DeepEqual(ret.end, getPrefix(ret.key))`?

So if someone specifies `WithPrefix` in `Watch` api, `end` will be non `nil`. And it *should* be equal to `getPrefix(ret.key)`. So I think it should be `!reflect.DeepEqual(ret.end, getPrefix(ret.key))`, not `reflect.DeepEqual(ret.end, getPrefix(ret.key))`. Test failed with `case ret.end != nil && reflect.DeepEqual(end, getPrefix(ret.key))`.

Thanks.